ir.go

```diff
@@ -508,6 +508,13 @@ func (m Module) AddNamedMetadataOperand(name string, operand Metadata) {
 	C.LLVMAddNamedMetadataOperand2(m.C, cname, operand.C)
 }
 
+func (m Module) AddModuleFlagUInt32(name string, value uint32) {
```

Comment:
There is a C API already for module flags, which is in fact more flexible as it uses Metadata objects. Can you use that instead?

https://llvm.org/doxygen/group__LLVMCCoreModule.html#ga91746975955737845b0f9a3b33c415f5